test/e2e-test.sh

```diff
@@ -0,0 +1,12 @@
+cd api
+
+failed=0
```

Comment:
can we add a shebang `#!/usr/bin/env bash` ?

Reply:
also the license

Reply:
Yep will add it 👍🏻